Gitlab/two-stage-pipeline-template/{{cookiecutter.outputDir}}/.gitlab-ci.yml

```diff
@@ -0,0 +1,149 @@
+variables:
+  SAM_TEMPLATE: {{cookiecutter.sam_template}}
+  AWS_ACCESS_KEY_ID_: ${AWS_ACCESS_KEY_ID}
+  AWS_SECRET_ACCESS_KEY_: ${AWS_SECRET_ACCESS_KEY}
+  TESTING_STACK_NAME: {{cookiecutter.testing_stack_name}}
+  TESTING_REGION: {{cookiecutter.testing_region}}
+  TESTING_PIPELINE_EXECUTION_ROLE: {{cookiecutter.testing_pipeline_execution_role}}
+  TESTING_CLOUDFORMATION_EXECUTION_ROLE: {{cookiecutter.testing_cloudformation_execution_role}}
+  TESTING_ARTIFACTS_BUCKET: {{cookiecutter.testing_artifacts_bucket}}
+  TESTING_ECR_REPO: {{cookiecutter.testing_ecr_repo}}
+  PROD_STACK_NAME: {{cookiecutter.prod_stack_name}}
+  PROD_REGION: {{cookiecutter.prod_region}}
+  PROD_PIPELINE_EXECUTION_ROLE: {{cookiecutter.prod_pipeline_execution_role}}
+  PROD_CLOUDFORMATION_EXECUTION_ROLE: {{cookiecutter.prod_cloudformation_execution_role}}
+  PROD_ARTIFACTS_BUCKET: {{cookiecutter.prod_artifacts_bucket}}
+  PROD_ECR_REPO: {{cookiecutter.prod_ecr_repo}}
+  DOCKER_DRIVER: overlay2
+  # Create the certificates inside this directory for both the server
+  # and client. The certificates used by the client will be created in
+  # /certs/client so we only need to share this directory with the
+  # volume mount in `config.toml`.
+  DOCKER_TLS_CERTDIR: "/certs"
+
+image: docker:19.03.15
+
+services:
+    - docker:19.03.15-dind
+
+before_script:
```

Comment:
should we add ja `apk add jq` as well? using it in the assume-role script to extract the credentials will be more readable than `awk`

Reply:
This is something I once discussed with @hoffa as well. Using `awk` does not require `jq` so one fewer dependency to install. As for the readability, I don't feel there is a big difference. I personally know `awk` more than `jq`, others might otherwise.

Reply:
Better keep it POSIX-compatible where possible, that way the script is meaningful without additional external dependencies as well.